speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/CHANGES.txt

```diff
@@ -187,27 +187,47 @@ Changed:
             racestopmenu.cpp
                 RmStopRaceMenu()
                     changed  AssistedHuman is a human
+                    removed controls and forcefeedback menu
             legacymenu/
                 CMakeList.txt
                     added    ResearcherMenu.h and .cpp
                     added    DataSelectionMenu.h and .cpp
                     added    backend in both ADD_SDLIB_LIBRARY and ADD_SDLIB_INCLUDEDIR
-                mainscreens/mainmenu.cpp
-                    included "DataSelectionMenu.h"
-                    added    GoBack()
-                    changed  MainMenuInit()
+                    removed player, graph, opengl, advanced, simu, ai, hostsettings configs
+                mainscreens
+                    /mainmenu.cpp
```

Comment:
Small detail, but this is how we have been formatting the directories, so change that for consistency (also update this for all your other changes)
```suggestion
                mainscreens/
                    mainmenu.cpp
```